xbmc/guilib/Texture.cpp

```diff
@@ -55,6 +56,7 @@ void CTexture::Allocate(unsigned int width, unsigned int height, unsigned int fo
   m_imageWidth = m_originalWidth = width;
   m_imageHeight = m_originalHeight = height;
   m_format = format;
+  unsigned int allocationFormat = format;
```

Comment:
I would rather see a sized type (`uint32_t`) for new code but no sweat here really.

Reply:
1. I've made the selection logic a bit simpler.
2. Done.